test/common/testlib.py

```diff
@@ -650,6 +650,9 @@ def logout(self):
                 self.click('#logout')
         self.expect_load()
 
+        # Log out can cause race with polkit request
+        self.allow_journal_messages(".*couldn't create polkit session subject: No session for pid.*")
```

Comment:
Not sure if that wouldn't better belong into allow_restart_messages()`, but it's fine. One of these days we need to clean this up anyway.